### PR TITLE
Add debug symbols for native code

### LINKED ISSUE #2991: Add debug info for native code
Add this to the gradle

```
android.buildTypes.release.ndk.debugSymbolLevel = 'FULL'
```

so that debug info is added to logs from C.

### PR DESCRIPTION
**Pull request by EmmanuelMess**
#### Issue tracker   

Fixes #2991

#### Automatic tests

- [ ] Added test cases
  
#### Manual tests
- [ ] Done  
  

#### Build tasks success  

Successfully running following tasks on local:
- [ ] `./gradlew assembledebug`
- [ ] `./gradlew spotlessCheck`

### CODE REVIEW

File: file_operations/build.gradle [hunk 1]
```diff
@@ -21,6 +21,7 @@ android {
         release {
             minifyEnabled true
             proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
+            ndk.debugSymbolLevel = 'FULL'
```
Comment (VishalNehra) [inline file_operations/build.gradle:24]: What's the size difference of bundle? I thought changing it to full increases the size with quite some margin.

**Review by VishalNehra — CHANGES_REQUESTED** [review]

File: file_operations/build.gradle (hunk 1 above)
Comment (EmmanuelMess) [reply file_operations/build.gradle:24]: > I thought changing it to full increases the size with quite some margin.

There is just a single function in c. I will check though.

Comment (EmmanuelMess) [reply file_operations/build.gradle:24]: [control.zip](https://github.com/TeamAmaze/AmazeFileManager/files/7608661/control.zip)
imagen


[symbols.zip](https://github.com/TeamAmaze/AmazeFileManager/files/7608685/symbols.zip)

imagen

**Comment by VishalNehra** [comment]
Sorry, i have no idea how I missed your reply here.

**Review by VishalNehra — APPROVED** [review]

**Pull request merged**